### PR TITLE
add task calendar  solution

### PR DESCRIPTION
**Pull request by booobryyyk**
[DEMO LINK](https://maksym-bobryk.github.io/layout_calendar/)
[TEST REPORT LINK](https://maksym-bobryk.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/styles/utils/variables.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+$block-size: 100px;
+$block-gap: 1px;
+$grey-color: #eee;
```
Comment (amashutikov) [inline src/styles/utils/variables.scss:3]: We are using this color only once. So i think there is no need to make separete variable for it.

**Review by amashutikov — APPROVED** [review]
Good job! Everything looks nice to me)

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,14 @@
+@import "./utils/variables.scss";
+@import "./utils/mixins.scss";
+@import "./blocks/calendar.scss";
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```
Comment (Anna-Tkachenko) [inline src/styles/main.scss:9]: Will be better to move it to some class, instead of setting all styles directly to `body` tag

**Review by Anna-Tkachenko — APPROVED** [review]
Well Done! ⭐️

**Pull request closed without merging**